buildenv

```diff
@@ -7,19 +7,40 @@ export ZOPEN_TARBALL_URL="https://ftp.gnu.org/gnu/gettext/gettext-0.21.tar.gz"
 export ZOPEN_TARBALL_DEPS="curl gzip tar make m4 perl"
 
 export ZOPEN_GIT_URL="https://github.com/autotools-mirror/gettext.git"
-export ZOPEN_GIT_DEPS="git make tar m4 perl autoconf automake help2man makeinfo xz"
+export ZOPEN_GIT_DEPS="git make tar m4 perl autoconf automake help2man makeinfo xz ncurses"
 
-export ZOPEN_EXTRA_CFLAGS="-qnocsect" # required because of multiple lock.c CUs
-export ZOPEN_EXTRA_LDFLAGS=""
-
-export ZOPEN_CHECK="skip"
+export ZOPEN_EXTRA_CONFIGURE_OPTS='--disable-acl --disable-ncurses'
 
 if [ "${ZOPEN_TYPE}x" = "TARBALLx" ]; then
 	export ZOPEN_BOOTSTRAP=skip
 fi
 
 zopen_check_results()
 {
-  return 3 # non-functional
+#
+# Not running many tests yet
+#
+# TOTAL: 2
+# PASS:  1
+# SKIP:  0
+# XFAIL: 0
+# FAIL:  1
+# XPASS: 0
+# ERROR: 0
+  dir="$1"
+  pfx="$2"
+  chk="$1/$2_check.log"
+
+  # Echo the following information to guage build health
```

Comment:
Minor typo `guage` -> `gauge`

Reply:
that's what i get for cut'n'paste :). will fix next time i crack this open